teachopencadd/talktorials/019_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1250 @@
+{

```

Comment:
<p>RCSB offers a simple way to get PDBs via URLs: <code>https://files.rcsb.org/download/3POZ.pdb</code>.  Do we need the <code>get_pdb_file</code>  function? We could use <code>requests</code>  instead. Up to you!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>